src/Selenium2Library/keywords/cookie.py

```diff
@@ -1,46 +1,58 @@
-from .keywordgroup import KeywordGroup
+from Selenium2Library.robotlibcore import keyword
```

Comment:
I think it would be better to use `from .robotlibrarycore import keyword` here. Makes it easier to change the library name if/when we decide to do it.